config/database.yml

```diff
@@ -2,16 +2,16 @@ development:
   adapter: mysql2
   encoding: utf8mb4
   database: hamers
-  username: hamers
-  password: hamers
+  username: root
+  password:
   host: 127.0.0.1
```

Comment:
`.gitignore`?

Reply:
Staat  ie al in, maar heeft ie niet door